sBTC SIP

```diff
@@ -0,0 +1,166 @@
+
+sBTC is a SIP-010 token on the Stacks blockchain, backed 1:1 against BTC, and operated by a decentralized set of signers. When BTC is locked on the Bitcoin L1, an equivalent amount of sBTC is issued on the Stacks layer, ensuring a consistent 1:1 ratio of sBTC:BTC. Users can redeem their sBTC at any time by submitting a withdrawal request. Once the request is processed by sBTC signers, BTC is returned to the user’s specified address on the Bitcoin L1.
+
+sBTC will have a crucial role in scaling Bitcoin, as well as introducing new and innovative functionalities to users, growing the size of the overall Bitcoin ecosystem. This SIP aims to describe the sBTC system, the process for signer selection, and the features available in the initial release compared to subsequent versions of sBTC. It does not attempt to describe the low-level technical details of any subsequent release, which will be provided in a future SIP.
+
+## Introduction
+
+## Glossary
+
+| Term                     | Definition |
+|--------------------------|------------|
+| SIP-10 Token             | A token on the Stacks blockchain that adheres to the fungible token standards outlined in SIP-10. |
+| sBTC                     | A SIP-10 token on the Stacks Blockchain that can be turned back into BTC on the Bitcoin Blockchain. 1 sBTC is equivalent to 1 BTC on the Bitcoin Blockchain. |
+| sBTC operation           | An operation that initiates some action from the sBTC protocol. |
+| Bitcoin holder           | A person or system holding BTC, interested in acquiring sBTC. |
+| sBTC holder              | A person or system holding sBTC, interested in turning it back into BTC. |
+| .sbtc contract           | A smart contract (or a collection of contracts) defining the sBTC token and functions related to it. |
+| sBTC UTxO                | The single UTxO holding the entire BTC balance that’s pegged into sBTC. This UTxO is managed and maintained by the sBTC Signers. |
+| Stacks Signer            | The standard Stacks Signer introduced in Nakamoto. |
+| sBTC Bootstrap Signer    | A completely separate signer from the Stacks signer that will sign sBTC operations and communicate with contracts on the chain to make that feasible. This entity is registered as an sBTC signer and has partial access to spending the sBTC UTxO. This only exists in sBTC-v1. In subsequent sBTC releases this functionality will be native in the Stacks Signer. |
+| sBTC Signer              | sBTC Signer is a more general term for the entity that signs sBTC transactions under any version of sBTC. Under sBTC-v1 the sBTC signer is the sBTC Bootstrap Signer which is wholly separate from the Stacks Signer, but in subsequent releases of sBTC the Stacks Signer will also be responsible for signing sBTC operations, so it would be the sBTC Signer. |
```

Comment:
It is not yet clear to me that Stacks signers are going to also be sBTC signers (and prescribing this now will have drastic follow-on consequences to the design of the "real" sBTC).  I for one do _not_ believe that the stacks signers are the same entities as the sBTC signers.

Reply:
+1 i think it's too soon to assume that the current stacks-signer *shall* be a sbtc signer. unless there are details about how this may happen later in the document (still reading), i would remove that line. 
secondly, I don't like the use of `sBTC-v1` unless there are plans to outline what this entails - will there be a v[2-9]* sip? should the abstract/title etc be changed to specifically reflect this?

one possible change would be `Under this SIP, the sBTC signer...` and leave off the v1 wording entirely.

Reply:
Previous design documents referred to Stackers as maintaining the peg wallet. I believe it was widely understood that Stacks Signers would also become sBTC signers. The idea that sBTC signers would be wholly separate from Stacks signers is relatively new.

Reply:
That is neither promised nor guaranteed in any way, shape, or form anywhere.  As mentioned here and elsewhere, the sBTC system that this SIP alludes to does not exist.  Therefore, this SIP should not be describing it.